### PR TITLE
chapter1: avoid derogatory term 'mess'

### PR DESCRIPTION
**Pull request by xypron**
Describe the intent of the EBBR specification using more positive terms.

### CODE REVIEW

File: source/chapter1-about.rst [hunk 1]
```diff
@@ -55,8 +55,9 @@ The challenge for EBBR is to define a set of boot standards that reduce the
 amount of custom engineering required, make it possible for OS distributions to
 support embedded platforms, while still preserving the firmware stack that
 product vendors are comfortable with.
-Or in simpler terms, EBBR is designed to solve the embedded boot mess by
-adding a defined standard (UEFI) to the existing firmware projects (U-Boot).
+Or in simpler terms, EBBR is designed to reduce the embedded boot heterogeneity
+by implementing a widely accepted standard (UEFI) in the existing firmware
+projects (U-Boot).
```
Comment (nullr0ute) [inline source/chapter1-about.rst:60]: You can drop "the" here and just have "in existing firmware projects" or s/projects/project/ because you explicitly mention a single project.

Otherwise LGTM.

File: source/chapter1-about.rst [hunk 2]
```diff
@@ -55,8 +55,9 @@ The challenge for EBBR is to define a set of boot standards that reduce the
 amount of custom engineering required, make it possible for OS distributions to
 support embedded platforms, while still preserving the firmware stack that
 product vendors are comfortable with.
-Or in simpler terms, EBBR is designed to solve the embedded boot mess by
-adding a defined standard (UEFI) to the existing firmware projects (U-Boot).
+Or in simpler terms, EBBR is designed to reduce the embedded boot heterogeneity
```
Comment (robherring) [inline source/chapter1-about.rst:58]: I would use 'variation' or 'differences' rather than heterogeneity. Might be easier for non-native English speaks?

**Comment by vstehle** [comment]
Thanks all for having a look.
@xypron, I can merge it with the modifications from @nullr0ute and @robherring if that helps, or do you want to respin it yourself?

**Comment by vstehle** [comment]
Merged with the modifications from @nullr0ute and @robherring.
